### PR TITLE
Minor refactoring and remove dependency on yiisoft/strings.

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | -

- Minor refactoring for arrayable names.
- Remove dependency on yiisoft/strings.

### CODE REVIEW

File: src/Html.php [hunk 1]
```diff
@@ -1641,4 +1637,14 @@ public static function escapeJsRegularExpression(string $regexp): string
 
         return $pattern;
     }
+
+    private static function setNameArrayable(string $name): string
```
Comment (samdark) [inline src/Html.php:1641]: It is not setting anything so `set` prefix is confusing.

File: src/Html.php [hunk 2]
```diff
@@ -1641,4 +1637,14 @@ public static function escapeJsRegularExpression(string $regexp): string
...
+    {
+        return substr($name, -2) !== '[]' ? $name . '[]' : $name;
+    }
+
+    private static function unsetNameArrayable(string $name): string
```
Comment (samdark) [inline src/Html.php:1646]: It is not setting anything so `set` prefix is confusing.

File: src/Html.php (hunk 1 above)
Comment (samdark) [reply src/Html.php:1641]: `getArrayableName`?

File: src/Html.php (hunk 2 above)
Comment (samdark) [reply src/Html.php:1646]: `getNonArrayableName`?

**Review by samdark — CHANGES_REQUESTED** [review]

File: src/Html.php (hunk 1 above)
Comment (vjik) [reply src/Html.php]: Done!

File: src/Html.php (hunk 2 above)
Comment (vjik) [reply src/Html.php]: Done!

**Review by rustamwin — APPROVED** [review]

**Review by samdark — APPROVED** [review]

**Pull request merged**